examples/batch_processing/src/getting_started_async_batch_processor.py

```diff
@@ -0,0 +1,21 @@
+from aws_lambda_powertools.utilities.batch import async_batch_processor, EventType, AsyncBatchProcessor
+from aws_lambda_powertools.utilities.data_classes.sqs_event import SQSRecord
+from aws_lambda_powertools.utilities.typing import LambdaContext
+
+
+async def async_record_handler(record: SQSRecord):
```

Comment:
np: what would be a realistic example here? maybe an async crawler?

As part of docs refactoring, we haven't reached Batch yet, we're trying to include more complete examples.

Example: https://awslabs.github.io/aws-lambda-powertools-python/2.4.0/core/tracer/#ignoring-certain-http-endpoints